packages/formation/js/accordion.js

```diff
@@ -34,39 +34,44 @@ const loadAccordionHandler = () => {
     .forEach(element => {
       element.addEventListener('click', e => {
         const target = e.target;
-        const other = Array.from(
-          element.getElementsByClassName('usa-accordion-button'),
-        ).filter(item => item !== target);
 
-        const multiSelectable = toBoolean(
-          element.getAttribute('aria-multiselectable'),
-        );
+        // Checks whether the button has a click event already assigned to it.
+        // Specifically React Components.
+        if (!target.onclick) {
+          const other = Array.from(
+            element.getElementsByClassName('usa-accordion-button'),
+          ).filter(item => item !== target);
 
-        if (target.getAttribute('aria-controls')) {
-          if (!multiSelectable) {
-            other.forEach(el => {
-              const contentEl = el.getAttribute('aria-controls');
+          const multiSelectable = toBoolean(
+            element.getAttribute('aria-multiselectable'),
+          );
 
-              el.setAttribute('aria-expanded', 'false');
+          if (target.getAttribute('aria-controls')) {
+            if (!multiSelectable) {
+              other.forEach(el => {
+                const contentEl = el.getAttribute('aria-controls');
 
-              document
-                .getElementById(contentEl)
-                .setAttribute('aria-hidden', 'true');
-            });
-          }
+                el.setAttribute('aria-expanded', 'false');
 
-          const dropDownElement = document.getElementById(
-            target.getAttribute('aria-controls'),
-          );
-          const targetExpandedAttr = toBoolean(
-            target.getAttribute('aria-expanded'),
-          );
+                document
+                  .getElementById(contentEl)
+                  .setAttribute('aria-hidden', 'true');
+              });
+            }
+
+            const dropDownElement = document.getElementById(
```

Comment:
are we sure that there will only ever be one `aria-controls` on the page?

Reply:
@rianfowler I'm not sure but under the target element there should only be 1. I guess the only situation where there might be another is a nested accordion. Do we want to try to handle this use case.

Reply:
I see now- we're getting the id for that control which is unique and using that to query the element